tests/kernels/test_moe.py

```diff
@@ -10,10 +10,10 @@
 from transformers.models.mixtral.modeling_mixtral import MixtralSparseMoeBlock
 
 from vllm.model_executor.layers.activation import SiluAndMul
-from vllm.model_executor.layers.fused_moe import (fused_marlin_moe, fused_moe,
-                                                  single_marlin_moe)
+from vllm.model_executor.layers.fused_moe import fused_moe, single_marlin_moe
+from vllm.model_executor.layers.fused_moe.fused_moe_marlin import fused_moe_marlin
```

Comment:
nit: I think it would be good to keep `single_marlin_moe` in the same place as `fused_moe_marlin`, even if the former is only used for testing